### ISSUE TITLE
Could anyone explain why don't use validation for baseline?

### ISSUE THREAD

**Issue by rainylt** [wyharveychen/CloserLookFewShot#61]
If we don't use validation and don't calculate accuracy during training, how can we know when to stop and how it performs in the training set, and whether it has overfitted?  Just stare at the loss?

**Comment by ashok-arjun** [comment]
With baseline, we cannot afford to train a linear classifier (the last layer) everytime we do a validation.

However with protonets, validation is just inference. That's why
